app/views/sidebar.ts

```diff
@@ -597,20 +597,24 @@ const resultsContent = (
         let orgClass = '';
         let orgName = '';
 
-        if (organizations.provinces[result.event.collection]) {
-            orgClass = 'province';
-            orgName = organizations.provinces[result.event.collection];
-        } else if (organizations.municipalities[result.event.collection]) {
-            orgClass = 'municipality';
-            orgName = 'Gemeente ' + organizations.municipalities[result.event.collection];
+        const match = result.index.match(/(ori|osi)_([a-z-]+)_\d+/);

```

Comment:
purely hypothetical, but o[rs]i is slightly quicker

Reply:
so hypothetical that on a test of 10.000.000 iterations it mattered a whooping 30-100ms